VisualPinball.Unity/VisualPinball.Unity.Editor/Layers/LayerEditor.cs

```diff
@@ -26,11 +32,34 @@ protected virtual void OnEnable()
 			if (_treeViewState == null)
 				_treeViewState = new TreeViewState();
 
-			_treeView = new LayerTreeView(_treeViewState);
+			if (_layerHandler== null)
+				_layerHandler = new LayerHandler();
+
+			if (_searchField == null)
+				_searchField = new SearchField();

```

Comment:
I'm not going to nag you anymore about this, but prefer curly brackets even around oneliners. They make diffs easier to read, and are less error-prone.

Reply:
sure :)